admin_interface/locale/it/LC_MESSAGES/django.po

```diff
@@ -175,10 +179,36 @@ msgstr "codice"
 msgid "display"
 msgstr "visualizzazione"
 
+#: admin_interface/models.py
+msgid "Default: #333 (required fields, error messages)"
```

Comment:
Please do not include the default color in the translated string (apply this change to all similar occurrences).